### PR TITLE
DOCSP-34346 - Expanded cursor example

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-34346
Staging - https://preview-mongodbmongokart.gatsbyjs.io/csharp/docsp-34346-print-cursor/fundamentals/crud/read-operations/retrieve/#example

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/retrieve.txt [hunk 1]
```diff
@@ -221,7 +228,7 @@ The following example performs these actions:
 - Finds all documents with "Pizza" in the ``cuisine`` field
 - Sets the ``BatchSize`` to ``3``
 - Stores the results in a cursor
-- Prints the number of documents currently held in the cursor
+- Prints the documents in the cursor
```
Comment (rustagir) [inline source/fundamentals/crud/read-operations/retrieve.txt:231]: S: small technical wording fix, as the cursor is not a data structure but references documents 
```suggestion
- Prints the documents referenced by the cursor
```

File: source/fundamentals/crud/read-operations/retrieve.txt [hunk 2]
```diff
@@ -221,7 +228,7 @@ The following example performs these actions:
 - Finds all documents with "Pizza" in the ``cuisine`` field
```
Comment (rustagir) [inline source/fundamentals/crud/read-operations/retrieve.txt:228]: S: cant comment there, but change wording to:
This example performs the following actions:

File: source/fundamentals/crud/read-operations/retrieve.txt [hunk 3]
```diff
@@ -238,13 +245,17 @@ The following example performs these actions:
```
Comment (rustagir) [inline source/fundamentals/crud/read-operations/retrieve.txt:245]: S: shorten this code example by removing the newlines above when instantiating the FindOptions

**Review by rustagir — APPROVED** [review]
lgtm with some small suggestions

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**